src/io/flutter/editor/FlutterPubspecNotificationProvider.java

```diff
@@ -29,6 +29,12 @@
 public class FlutterPubspecNotificationProvider extends EditorNotifications.Provider<EditorNotificationPanel> implements DumbAware {
   private static final Key<EditorNotificationPanel> KEY = Key.create("flutter.pubspec");
 
+  @NotNull private final Project project;
+
+  public FlutterPubspecNotificationProvider(@NotNull Project project) {
+    this.project = project;
```

Comment:
Since we're extending `AbstractProjectComponent` I think we can just do `super(project)` (and update access below to `myProject`)?